tests/main/views/test_suppliers.py

```diff
@@ -763,26 +763,16 @@ class TestSetSupplierDeclarations(BaseApplicationTest, FixtureMixin, JSONUpdateT
 
     def setup(self):
         super(TestSetSupplierDeclarations, self).setup()
-        with self.app.app_context():
-            framework = Framework(
-                slug='test-open',
-                name='Test open',
-                framework='g-cloud',
-                status='open')
-            db.session.add(framework)
-            db.session.commit()
+        framework = Framework(
+            id=100,
```

Comment:
Is it worth putting an inline comment here saying no teardown needed as IDs >= 100 get torn down automatically?

Reply:
Done 👍